### PR TITLE
[DP-1832] Update maven checkstyle plugin

### PR DESCRIPTION
**Pull request by elroyalva**
Maven checkstyle depends on puppycrawl checkstyle 6.9. To bump up puppcrawl checkstyle we must also bump up maven checkstyle

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -57,7 +57,7 @@
         <killbill-api.version>0.50.2</killbill-api.version>
 
         <osgi.version>5.0.0</osgi.version>
-        <version.checkstyle>2.16</version.checkstyle>
+        <version.checkstyle>3.1.0</version.checkstyle>
```
Comment (Pepelo) [inline pom.xml]: Do we have breaking changes?

Comment (elroyalva) [reply pom.xml]: Nothing breaks in killbill-authnet-plugin and killbill-resources. Testing with billing-service and swiper-service will take some time

Comment (elroyalva) [reply pom.xml]: Also, I think this is a checkstyle plugin and _probably_ only used at build time. I don't think it should affect anything else.

File: pom.xml [hunk 2]
```diff
@@ -730,7 +730,7 @@
                         <instructions>
                             <!-- Property bundle.activator.class is defined in individual plugins -->
                             <Bundle-Activator>${bundle.activator.class}</Bundle-Activator>
-                            <Export-Package/>
+                            <Export-Package />
```
Comment (Pepelo) [inline pom.xml]: why this change?

Comment (elroyalva) [reply pom.xml]: It was added by IntelliJ. Not sure why. But let me take it out
